### PR TITLE
fixed report generation in cucumber environment

### PR DESCRIPTION
**Pull request by dpoetzsch**
## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other

## What is the current behavior?

In a cucumber environment only empty reports are generated.

## What is the new behavior?

In a cucumber environment reports are generated as they should.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

I also added the field `suiteName` to the report. For cucumber this would be the name of the feature, for mocha/jasmine this could be the name of the `describe` (not implemented though).

### CODE REVIEW

File: packages/reporter/interfaces.d.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ interface MatchCommandResult {
 export type TestContextResult = Omit<ReportData, 'matchers'>;
 
 export interface ReportData {
+  suiteName?: string;
```
Comment (ennjin) [inline packages/reporter/interfaces.d.ts]: I guess we should have same structure for all test frameworks. How I can see there is two ways to solve it:
1. Implement `suiteName` for jasmine and mocha too in this PR
2. Remove `suiteName` and implement this one for all test frameworks in new PR

What do you think about this?

Comment (dpoetzsch) [reply packages/reporter/interfaces.d.ts]: I thought you might comment on that :) 

I did a quick research. At least with jasmine we only get the following:

```javascript
{
  id: 'spec0',
  description: 'Should check element',
  fullName: 'Visual regression Should check element',
  failedExpectations: [],
  passedExpectations: [
    {
      matcherName: 'toEqual',
      message: 'Passed.',
      stack: '',
      passed: true
    }
  ],
  deprecationWarnings: [],
  pendingReason: '',
  duration: 219,
  start: 1584043081114,
  title: 'Should check element',
  fullTitle: 'Visual regression Should check element',
  error: undefined,
  passed: true
}
```

As there is no structural info for the suite I implemented a quick option to extract it from the fullTitle. It works for jasmine, I guess it does as well for mocha.

Comment (ennjin) [reply packages/reporter/interfaces.d.ts]: There may be case when describe will be nested, for example.
```ts
describe('Level 1', () => {
  describe('Level 2', () => {
    it('Some test case here', () => {})
  })
})
```
So, `fullTitle` will be `Level 1 Level 2 Some test case here` and it's not expected behaviour. I guess we should create new PR and discuss the possible solutions.

Comment (dpoetzsch) [reply packages/reporter/interfaces.d.ts]: Done.

**Comment by ennjin** [comment]
Now looks good to me, thanks!

**Pull request merged**